UNITTESTS/mbed_unittest.py

```diff
@@ -71,9 +71,8 @@ def _mbed_unittest_test(options, cwd, pwd):
                                  clean=options.clean)
 
     if options.compile_only:
-        # Create makefiles
-        src_path = os.path.relpath(pwd, options.build)
-        tool.create_makefiles(path_to_src=src_path,
+        # Create makefiles        
+        tool.create_makefiles(path_to_src=cwd,
```

Comment:
Why is this changed?

Reply:
The previous unit test build is also based on CMake and the build starts from `mbed-os/UNITTESTS` directory (Present working directory) but with the new changes build starts from the mbed-os root directory (Current working directory) so backward compatibility to support mbed CLI 1 just tweak here to start the build from the current working directory (mbed-os root dir) and we may remove this mbed-os/UNITTESTS/*.py module once deprecate CLI1 as we can able add this Cmake command calling directly into mbed CLI2

Reply:
Cool, thanks for the explanation!